README.md

```diff
@@ -1,6 +1,8 @@
 # MetaMask Browser Extension
 [![Build Status](https://circleci.com/gh/MetaMask/metamask-extension.svg?style=shield&circle-token=a1ddcf3cd38e29267f254c9c59d556d513e3a1fd)](https://circleci.com/gh/MetaMask/metamask-extension) [![Coverage Status](https://coveralls.io/repos/github/MetaMask/metamask-extension/badge.svg?branch=master)](https://coveralls.io/github/MetaMask/metamask-extension?branch=master) [![Greenkeeper badge](https://badges.greenkeeper.io/MetaMask/metamask-extension.svg)](https://greenkeeper.io/) [![Stories in Ready](https://badge.waffle.io/MetaMask/metamask-extension.png?label=in%20progress&title=waffle.io)](https://waffle.io/MetaMask/metamask-extension)
 
+🚨 As of 7/25/18, the MetaMask extension has been removed from the Chrome Web Store. To download the latest version of MetaMask, use the [4.8 Chrome build](https://github.com/MetaMask/metamask-extension/releases/download/v4.8.0/metamask-chrome-4.8.0.zip) (also available on our [Releases](https://github.com/MetaMask/metamask-extension/releases) list), and load it on the `chrome://extensions` page. For more detailed steps, visit our [help center](https://consensys.zendesk.com/hc/en-us/articles/360004134152-How-to-Install-MetaMask-Manually). Follow [@metamask_io](https://twitter.com/metamask_io) on Twitter for updates. 🚨
```

Comment:
Rather than link to a specific release, maybe just say "you can find the latest release at the top of the releases page", so we don't have to update this with every release?